src/lib/looper.js

```diff
@@ -20,11 +20,8 @@ export default function(opts) {
 
     async function loop() {
         const timeStart = new Date().getUTCMilliseconds();
-        try {
```

Comment:
removing this was a bad idea. just did it because of it being annoying to test :grimacing: